src/test/java/org/apache/pirk/serialization/SerializationTest.java

```diff
@@ -75,6 +94,106 @@ public void testJavaSerDe() throws Exception
     Assert.assertTrue(deserializedDummyObject.equals(dummyRecord));
   }
 
+  @Test
+  public void testQuerierResponseSerializationDeserialization()
+  {
+    testQuerierResponseSerializationDeserialization(jsonSerializer);
+    testQuerierResponseSerializationDeserialization(javaSerializer);
```

Comment:
That line isn't in there twice. It's in there once with the arg jsonSerializer, and once with the arg javaSerializer. ;) Thus are both extant serialization services tested.